tests/nnstreamer_filter_custom/python/nns_tensor.py

```diff
@@ -0,0 +1,68 @@
+''' DO NOT CHANGE THIS FILE '''
+
+import enum
+import numpy as np
+
+NNS_TENSOR_RANK_LIMIT = 4
+
+#gst/nnstreamer/tensor_typedef.h
```

Comment:
Please append annotations in front of the class and function to generate Doxygen-based document automatically.
* Reference: https://github.com/nnsuite/nnstreamer/blob/master/Documentation/doxygen-documentation.md#case-study-python